src/main/java/com/elixr/poc/common/util/MessagesUtil.java

```diff
@@ -0,0 +1,50 @@
+package com.elixr.poc.common.util;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import java.text.MessageFormat;
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+
+@Slf4j
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class MessagesUtil {
+
+    private static final String MESSAGE_PROPERTY_FILE_PREFIX = "messages";
+
+    private static ResourceBundle moduleMessagesResourceBundle;
+
+    static {
+        try {
+            // Initialize messagesResourceBundle in a static block because the module may not have messages.properties.
+            moduleMessagesResourceBundle = ResourceBundle.getBundle(MESSAGE_PROPERTY_FILE_PREFIX);
+        } catch (MissingResourceException e) {
+            moduleMessagesResourceBundle = null;
+        }
+    }
+
+    public static String getMessage(String key, Object... arguments) {
```

Comment:
Ca you explain what does Object... means?

Reply:
Object... is used for multiple arguments to be passed.